cunumeric/random/random.py

```diff
@@ -28,7 +28,14 @@
     from ..types import NdShapeLike
 
 
+default_rng = generator.default_rng
+
+
 def seed(init: Union[int, None] = None) -> None:
+    """
+    Ressed the legacy random number generator.
+    This function is effective only in the CPU-only binary.
```

Comment:
Also if cuRAND is included (which AFAIU can also be the case in a CPU-only build)?

Reply:
Updated the docstring. This function doesn't do anything when cuRAND is used, so that needs to be fixed first.